lib/standalone.js

```diff
@@ -30,7 +30,7 @@ const writeFileAtomic = require('write-file-atomic');
  */
 async function standalone({
 	allowEmptyInput = false,
```

Comment:
Oh, tests fail. This change also is needed:

```suggestion
	allowEmptyInput,
```